packages/optimizely-sdk/lib/utils/user_id_validator/index.tests.js

```
@@ -45,6 +45,24 @@ describe('lib/utils/user_id_validator', function() {
           userIdValidator.validate('');
         }, sprintf(ERROR_MESSAGES.INVALID_USER_ID, 'USER_ID_VALIDATOR'));
       });
+
+      it('should throw an error if userId is int', function() {
```

These can be consolidated into one test case, which is, `should throw an error if userId is not a string` and you can run the 3 assertions in that one test